stylesheets/manageList.less

```diff
@@ -5,12 +5,17 @@
   }
 
   th.action, td.action {
-    width: 160px;
     a {
       color: black;
     }
   }
 
+  .tags {
+    th.action, td.action {
+      width: 200px;
+    }
+  }
```

Comment:
Consider restoring the original 160px value instead of removing it entirely.

Reply:
Fixed :+1: